docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -590,6 +590,105 @@ Loading cert/key pair from "/var/lib/kubelet/pki/kubelet-client-current.pem
 "Failed to connect to apiserver" err="Get \"https://10.150.223.2:6443/healthz?timeout=1s\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"
 ```
 
+### Troubleshooting DNS issues in EKSA cluster
+
+DNS resolution is managed by coreDNS, a DNS server deployed as a pod within the cluster, it helps resolve service names to corresponding IP Addresses. If DNS resolution fails, applications maybe unable to connect to services or external endpoints.
+
+#### Common errors related to DNS issues:
+- Pods are unable to resolve the DNS names of other services.
```

Comment:
this is one issue customer can face, you should also add the issue that we noticed- where a workload cluster could not be created as coredns was not connected to correct nameserver.